catalog/models.py

```diff
@@ -4,6 +4,71 @@
 from django.db import models
 from common.models import Student
 
+class Attribute:
+    """Class that describes an attribute.
```

Comment:
This is a lot better. Stylistically, a few minor things:
1. Move method documentation into the method itself.
2. Name all variables and methods consistently with the rest of the codebase: `foo_bar`, not `foobar` or `fooBar`.